Mathlib/RingTheory/Localization/Basic.lean

```diff
@@ -1090,7 +1090,9 @@ section
 variable [Algebra Rₘ Sₘ] [Algebra R Sₘ] [IsScalarTower R Rₘ Sₘ] [IsScalarTower R S Sₘ]
 variable (S Rₘ Sₘ)
 
-theorem IsLocalization.map_units_map_submonoid (y : M) : IsUnit (algebraMap R Sₘ y) := by
+theorem IsLocalization.map_units_map_submonoid
+    [i: IsLocalization (Algebra.algebraMapSubmonoid S M) Sₘ] (y : M) :
```

Comment:
Pre-existing, but: can you remove the name `i`, please? We prefer to not pass such typeclasses as named arguments.
```suggestion
    [IsLocalization (Algebra.algebraMapSubmonoid S M) Sₘ] (y : M) :